internal/pubsub/pubsub.go

```diff
@@ -30,12 +30,13 @@ type Message struct {
 	// Metadata has key/value metadata for the message.
 	Metadata map[string]string
 
-	// AckID identifies the message on the server.
```

Comment:
Why is this stuff in the conformance test PR?

Reply:
I agree it should not be here. The original motivation was that when a conformance test was failing, the diff output was too spammy because of the Subscription pointer in Message. Anyway, I'm going to split that change out into a separate PR for clarity.